Library/brew.rb

```diff
@@ -129,17 +129,17 @@ def require?(path)
   $stderr.puts # seemingly a newline is typical
   exit 130
 rescue BuildError => e
-  report_analytics_exception(e)
+  Utils::Analytics::report_exception(e)
```

Comment:
Looks like it should have been `Utils::Analytics.report_exception` here and in the two instances below.